static/js/publisher/market/storageCommands.js

```diff
@@ -0,0 +1,30 @@
+function storageCommands(
+  e,
+  formEl,
+  snap_name,
+  ignoreChangesOnUnload,
+  context = window
+) {
+  const key = `${snap_name}-command`;
+  if (e.key === key) {
+    switch (e.newValue) {
+      case "edit":
+        context.localStorage.removeItem(key);
```

Comment:
As it seems it is always removed from localStorage could it be just moved out from `switch`?